.github/workflows/testing.yml

```diff
@@ -0,0 +1,42 @@
+name: Unit Tests
+
+on:
+  push:
+  pull_request:
+  schedule:
+      - cron: '00 4 * * *'  # daily at 4AM
```

Comment:
Do we need that frequent testing? Maybe weekly would make sense.

Reply:
A side note: I really like that website to validate my crontab entries: [`https://crontab.guru/#00_4_*_*_*`](https://crontab.guru/#00_4_*_*_*).

Reply:
We do daily currently to quickly pick up problems like breakages upstream. I could be worth re-evaluating but let's do that consistently across all projects.